templates/init/hosting/index.html

```diff
@@ -10,13 +10,18 @@
     <!-- include only the Firebase features as you need -->
     <script defer src="/__/firebase/{{VERSION}}/firebase-auth.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-database.js"></script>
+    <script defer src="/__/firebase/{{VERSION}}/firebase-firestore.js"></script>
+    <script defer src="/__/firebase/{{VERSION}}/firebase-functions.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-messaging.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-storage.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-analytics.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-remote-config.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-performance.js"></script>
-    <!-- initialize the SDK after all desired features are loaded -->
-    <script defer src="/__/firebase/init.js"></script>
+    <!-- 
+      initialize the SDK after all desired features are loaded, set useEmulator to false
+      to avoid connecting the SDK to running emulators.
+    -->
+    <script defer src="/__/firebase/init.js?useEmulator=true"></script>
```

Comment:
I'm not sure `useEmulator=true` here will be okay. What happens when this file is deployed to a production (on real Hosting) environment? Well, I guess nothing, right? That `init.js` knows nothing about this... is this functionally a noop?

Reply:
Yeah in production to my eyes it looks like Hosting just ignores it ... but I was hoping you could confirm that by looking into the implementation (or tell me this is a bad idea and I am relying on something I shouldn't)

Reply:
@bkendall I added some tests to the hosting integration test to verify this, so at least if production breaks this we'd have a canary.